app/javascript/chat/chat.jsx

```diff
@@ -1649,40 +1649,44 @@ export default class Chat extends Component {
         id="message"
         className={
           showDeleteModal
-            ? 'message__delete__modal'
-            : 'message__delete__modal message__delete__modal__hide'
+            ? 'message__delete__modal crayons-modal crayons-modal--s absolute'
+            : 'message__delete__modal message__delete__modal__hide crayons-modal crayons-modal--s absolute'
         }
       >
-        <div className="modal__content">
-          <h3> Are you sure, you want to delete this message ?</h3>
-
-          <div className="delete__action__buttons">
-            <div
-              role="button"
-              className="message__cancel__button"
-              onClick={this.handleCloseDeleteModal}
-              tabIndex="0"
-              onKeyUp={(e) => {
-                if (e.keyCode === 13) this.handleCloseDeleteModal();
-              }}
-            >
-              {' '}
-              Cancel
-            </div>
-            <div
-              role="button"
-              className="message__delete__button"
-              onClick={this.handleMessageDelete}
-              tabIndex="0"
-              onKeyUp={(e) => {
-                if (e.keyCode === 13) this.handleMessageDelete();
-              }}
-            >
-              {' '}
-              Delete
+        <div className="crayons-modal__box">
+          <div className="crayons-modal__box__body">
+            <h3>
+              Are you sure, you want to delete this message ?
+            </h3>
+            <div className="delete-actions__container">
+              <div
```

Comment:
I'd like to update this `div` to be a `button` because that will be more semantic, but I'm nervous about breaking some test...

Reply:
definitely in favor of that, I can help out with the change, let me know :)

Reply:
@rhymes Can you teach me how to do this?

This is something I notice across different places when I'm updating the frond-end, and I'd like to figure out how to do this in the future too :)